### PR TITLE
Whm traits

### PR DESCRIPTION
**Pull request by nfaletra**
Implemented Divine Benison
Added Shield Def. Bonus traits

Also fixed a spot in mob_spell_container that was doing a redundant check.

### CODE REVIEW

File: sql/traits.sql [hunk 1]
```diff
@@ -479,6 +481,11 @@ INSERT INTO `traits` VALUES ('110','mag. burst bonus','13','90','2','487','7','A
 INSERT INTO `traits` VALUES ('110','mag. burst bonus','20','79','1','487','5','ABYSSEA','0');
 INSERT INTO `traits` VALUES ('110','mag. burst bonus','20','89','2','487','7','ABYSSEA','0');
 INSERT INTO `traits` VALUES ('110','mag. burst bonus','20','99','3','487','9','ABYSSEA','0');
+INSERT INTO `traits` VALUES ('111','divine benison','3','50','1','909','10','ABYSSEA','0');
```
Comment (davismj) [inline sql/traits.sql]: ModId be 910, right?

Numbers should not be quoted, see https://github.com/DarkstarProject/darkstar/blob/master/CONTRIBUTING.md#sql

Comment (nfaletra) [reply sql/traits.sql]: Oh yes. Forgot to update it after quick magic got merged.

I agree that numbers should not be in quotes but someone added them to the entirety of traits.sql. I lean towards consistency in these cases and I'm not going to remove them from the entire file and mess up the diff. I plan to clean up the whole file as a separate commit.

File: src/map/mob_spell_container.cpp [hunk 2]
```diff
@@ -67,7 +67,7 @@ void CMobSpellContainer::AddSpell(SpellID spellId)
         m_damageList.push_back(spellId);
 
     }
-    else if(spell->isNa() || spellId == SpellID::Erase){
+    else if(spell->isNa()){
```
Comment (Hozu) [inline src/map/mob_spell_container.cpp]: If you're removing Erase from the isNa() function, please add Erase check for Divine Veil here:
https://github.com/DarkstarProject/darkstar/blob/0807c177c8b4202d073c8ff54a79145d68c538fe/src/map/utils/battleutils.cpp#L4521

Though I'm not sure what the devs would prefer.

Comment (nfaletra) [reply src/map/mob_spell_container.cpp]: isNa() checks for Erase as part of the function call. I didn’t actually remove anything except redundant code.

Comment (nfaletra) [reply src/map/mob_spell_container.cpp]: isNa() checks for Erase as part of the function call. I didn’t actually remove anything except redundant code.

Comment (Hozu) [reply src/map/mob_spell_container.cpp]: Ooh, I missed that. My bad.

File: sql/traits.sql (hunk 1 above)
Comment (TeoTwawki) [reply sql/traits.sql]: After all the current prs changing that table are done I'm reformatting it with a key for sort purposes and tonstrip the quotes off the numbers. Only holding off so you guys don't have to rebase. Just a heads up.

Comment (nfaletra) [reply sql/traits.sql]: I'm holding off on more traits fixes for the time being after this one for a bit.

**Pull request merged**